Source/Private/ASTextNode2.mm

```diff
@@ -488,7 +488,54 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
    inAdditionalTruncationMessage:(out BOOL *)inAdditionalTruncationMessageOut
                  forHighlighting:(BOOL)highlighting
 {
-  AS_TEXT_ALERT_UNIMPLEMENTED_FEATURE();
+  ASDN::MutexLocker l(__instanceLock__);
+
+  ASTextContainer *containerCopy = [_textContainer copy];
+  containerCopy.size = self.calculatedSize;
+  ASTextLayout *layout = [ASTextNode2 compatibleLayoutWithContainer:containerCopy text:_attributedText];
```

Comment:
@Adlai-Holler unfortunately, without these, link handling does not work. I could try removing just one or the other, but instead I think it is going to require a more detailed look at what is going on.